bin/create-component

```diff
@@ -7,8 +7,6 @@ component_name=$1
 component_sub_path=$2
 component_path="${components_path}${component_sub_path:+/$component_sub_path}/$1"
 
-echo $component_path
-
```

Comment:
Ah yeah, I'd left that there on accident 😔

Reply:
I actually thought that this was mine, coz I've also has an exit below for investigative purposes :)